test/feature-erc20.js

```diff
@@ -52,7 +52,7 @@ beforeEach(async function () {
     receiver6,
     challenger2,
     sender7,
-    receiver7
+    receiver7,
```

Comment:
Remove this file of this commit.

Reply:
I removed the test command for the CI yml file, but you want to cancel the feature-erc20.js formatting ?

Reply:
Nop just don't commit this file.